Detectors/TRD/workflow/src/TRDGlobalTrackingSpec.cxx

```diff
@@ -144,13 +144,17 @@ void TRDGlobalTracking::run(ProcessingContext& pc)
   // load ITS-TPC matched tracks
   for (int iTrk = 0; iTrk < mChainTracking->mIOPtrs.nTracksTPCITSO2; ++iTrk) {
     const auto& trkITSTPC = mChainTracking->mIOPtrs.tracksTPCITSO2[iTrk];
-    GPUTRDTrack trkLoad(trkITSTPC, mTPCVdrift);
+    GPUTRDTracker::HelperTrackAttributes trkAttribs;
+    trkAttribs.mTime = trkITSTPC.getTimeMUS().getTimeStamp();
+    trkAttribs.mTimeAddMax = trkITSTPC.getTimeMUS().getTimeStampError();
+    trkAttribs.mTimeSubMax = trkITSTPC.getTimeMUS().getTimeStampError();
```

Comment:
@davidrohr I would like to add this option to the TRD reco options in GPUSettingsList.h via:
`AddOptionRTC(nSigmaTerrITSTPC, float, 4.f, "", 0, "Number of sigmas for ITS-TPC track time error estimate")`
Then I should be able to set the parameter from command line via `--configKeyValues "GPU_rec_trd.nSigmaTerrITSTPC=4;"` right?
And how could I access the option in TRDGlobalTrackingSpec? Should I just create a GPUSettingsRec object and take the option from there?

Reply:
The settings should be fetched automatically during
```
mRec->SetSettings(o2::base::Propagator::Instance()->getNominalBz(), &cfgRecoStep);
```
So they will be available for the TRDTracker.
In case you need to access them in the spec directly, you can use mRec->GetProcessingSettings()

Reply:
Thanks, I have added the option now